stable/database/values.yaml

```diff
@@ -351,6 +351,10 @@ database:
     #Some clusters require longer readiness probe timeouts
     readinessTimeoutSeconds: 5
   
+  #These annotations will pass throgh to the pod as supplied, useful for integrating 3rd party products such as Vault.
```

Comment:
nit: we usually have a space between `#` and the next word.

Reply:
Thanks, fixed